e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Where will this template be used?

Reply:
It will be used by the test deployment, should I move it to test folder?

Reply:
I also created this file in another PR#2064